api/src/test/java/com/datastrato/gravitino/TestNameIdentifier.java

```diff
@@ -41,9 +41,10 @@ public void testCreateWithInvalidArgs() {
     assertThrows(IllegalArgumentException.class, () -> NameIdentifier.of("a", null));
     assertThrows(IllegalArgumentException.class, () -> NameIdentifier.of("a", ""));
 
+    Namespace empty = Namespace.empty();
     assertThrows(IllegalArgumentException.class, () -> NameIdentifier.of(null, "a"));
-    assertThrows(IllegalArgumentException.class, () -> NameIdentifier.of(Namespace.empty(), null));
-    assertThrows(IllegalArgumentException.class, () -> NameIdentifier.of(Namespace.empty(), ""));
+    assertThrows(IllegalArgumentException.class, () -> NameIdentifier.of(empty, null));
```

Comment:
Should we verify the error message to confirm that it is the expected error?

Reply:
I'm not 100% sure what you mean. Checking error strings in tests can be fragile if you are suggesting that.

Reply:
For example:
```java
Exception exception =
        assertThrows(
            IllegalArgumentException.class, () -> NameIdentifier.of(Namespace.empty(), null));
    assertTrue(
        exception.getMessage().contains("Cannot create a NameIdentifier with null or empty name"),
        "Message: " + exception.getMessage());
```

Reply:
This PR was just a refactor to fix how assertThrows was misused; if you think we should check for message text, I suggest you raise another issue for that.

Reply:
OK, current changes LGTM.